server/middlewares/index.js

```diff
@@ -0,0 +1,7 @@
+/** Middlewares */
+import { validateUserSignUp } from './authValidation';
+
+/** validate user signup */
+export const validateSignUp = validateUserSignUp;
```

Comment:
I don't think you need this file. I believe you can call validateUserSignUp directly as middleware on any route